tests/test_coverage.test.ts

```diff
@@ -10,6 +10,7 @@ const excludedLinters: string[] = [
   "cspell",
   "nancy",
   "oxipng",
+  "sort-package-json",
```

Comment:
re: your comment in the description about testing. I'm planning to add some better helper methods for more manual trunk check runs (this is going to be particularly necessary for triggers, etc.). I will loop back to this linter then and add some tests for it.

Reply:
This will also require some slight modifications to the snapshot format (or at least a new naming schema for those types of tests).